Scripts/AreaLightTest.bv.lua

```diff
@@ -154,7 +154,7 @@ do
         materialSetupFunction()
         
         IdleFrames(1) 
-        CaptureScreenshot(g_screenshotOutputFolder .. '/' .. lightName .. '_' .. materialName ..'.png')
+        CaptureScreenshot(g_testCaseFolder .. '/' .. lightName .. '_' .. materialName ..'.png')
```

Comment:
Since you already have SetScreenshotsFolder and SetTestEnvFolder, I wonder if we should also add SetTestCaseFolder instead of making each script append it like this.

Reply:
I think inner folder structure should be managed locally. It can have multiple levels of test case, for example, it is also reasonable to add RHI/RPI as another level of test case.